### PR TITLE
shared styling v2

### PR DESCRIPTION
**Pull request by pritipsingh**

### CODE REVIEW

File: src/components/AliasSetup.js [hunk 1]
```diff
@@ -21,26 +21,26 @@ const AliasSetup = () => {
...
-        size="16px"
+        fontSize="16px"
         textTransform="none"
-        weight="500"
+        fontWeight="500"
       >
         This usually takes around 5 minutes.
-      </Span>
+      </SpanV2>
       <Item display="flex" direction="row" align="center" margin="50px 0px">
```
Comment (abhishek-01k) [inline src/components/AliasSetup.js:44]: We can change the Item in sharedStyling to ItemHV2 or ItemVV2 from sharedStylingV2. See if that can be done.

File: src/components/ChannelDetails.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { useNavigate } from 'react-router-dom';
 import { MdRemoveCircleOutline } from 'react-icons/md';
 
 // Internal Compoonents
```
Comment (abhishek-01k) [inline src/components/ChannelDetails.js:12]: In this file, Button is used from sharedstyling. Change it to ButtonV2. See at line no. 610.

File: src/structure/Header.tsx [hunk 3]
```diff
@@ -28,7 +28,7 @@ import { ReactComponent as EPNSLogoLight } from './assets/epnsLight.svg';
```
Comment (abhishek-01k) [inline src/structure/Header.tsx:28]: Same in this file Also Change the components imported from sharedstyling to SharedStylingV2. There are components being used like : Item, ItemH and Section.

**Comment by HarshRajat** [comment]
closing as stale

**Pull request closed without merging**